sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/EntityBindingLiveTests.cs

```diff
@@ -1223,10 +1223,15 @@ public async Task Call([Table(TableNameExpression)] IAsyncCollector<TableEntity>
             {
                 for (int i = 0; i < TableEntityWriter.MaxBatchSize * 4; i++)
                 {
-                    await collector.AddAsync(new TableEntity(PartitionKey, i.ToString())
+                    try
                     {
-                        ["Value"] = i
-                    });
+                        await collector.AddAsync(new TableEntity(PartitionKey, i.ToString()) { ["Value"] = i });
+                    }
+                    catch (FunctionInvocationException ex) when (ex.InnerException is TableTransactionFailedException ttfe && ttfe.Status == 429)
+                    {
+                        await Task.Delay(3000);
```

Comment:
Does this potentially run afoul of the time limit that you added in the test framework, or no, since we'll only trigger this when running against live resources?

Reply:
Yeah, it's LiveOnly, so the timeout doesn't apply. But that said, this should be calling Recording.Delay instead so that if this ever does run as a recorded test, it will not wait in real time.